.github/workflows/check_lint.yml

```diff
@@ -5,51 +5,56 @@ on:
       - oss
 
```

Comment:
I would also expect `.github/workflows/test.yml` to be changed so that `cargo test --no-default-feature` is run in the CI